### PR TITLE
New package: owncloudclient-2.0.1

### PR DESCRIPTION
**Pull request by samuelchodur**

### CODE REVIEW

File: srcpkgs/owncloudclient/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+hostmakedepends="neon-devel qtkeychain-devel sqlite-devel cmake qt-devel"
+makedepends="${hostmakedepends}"
+# i am just cheating here, not really sure what depends this package has
+# i was able to install and run owncloudcmd on fresh install of void.
+depends="python"
+short_desc="Connect to ownCloud servers"
+maintainer="Samuel Chodur <samuelchodur@gmail.com>"
+license="GPL-2"
+homepage="http://www.owncloud.org"
+distfiles="https://download.owncloud.com/desktop/stable/owncloudclient-2.0.1.tar.xz"
```
Comment (Gottox) [inline srcpkgs/owncloudclient/template]: Use `distfiles="https://download.owncloud.com/desktop/stable/${pkgname}-${version}.tar.xz"`

File: srcpkgs/owncloudclient/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'owncloudclient'
+pkgname="owncloudclient"
+version=2.0.1
+revision=1
+only_for_archs="i686 x86_64"
+build_style=cmake
+hostmakedepends="neon-devel qtkeychain-devel sqlite-devel cmake qt-devel"
+makedepends="${hostmakedepends}"
+# i am just cheating here, not really sure what depends this package has
+# i was able to install and run owncloudcmd on fresh install of void.
```
Comment (Gottox) [inline srcpkgs/owncloudclient/template]: Cheating, he? :cry: 

A rule of thumb `-devel` packages go into makedepends, everything else in hostmakedepends. This is not always true, but should be good enough.

**Comment by Gottox** [comment]
Please use proper commit messages. For this PR please join both commits using `git rebase -i master; <add fixup to the latter>; git push -f`

**Comment by samuelchodur** [comment]
thanks for taking the time to help me out. I hope I did everything I needed. I am new with making template files and also using git.

**Pull request closed without merging**